docs/running-on-yarn.md

```diff
@@ -495,6 +468,20 @@ To use a custom metrics.properties for the application master and executors, upd
   name matches both the include and the exclude pattern, this file will be excluded eventually.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.report.interval</code></td>
```

Comment:
I am not sure whether these are meant to be exposed to users -- not secret exactly, but not sure if they're to be advertised. Maybe it's fine to document. I'm neutral.